File: Ruby/enemy.rb
class Enemy
  attr_reader :lv
  attr_accessor :hp
  attr_reader :defense
  attr_reader :attack
  attr_reader :hit
  attr_reader :flee
  attr_reader :name

  def initialize(lv = 0, hp = 0, defense = 0, attack = 0, hit = 0, flee = 0, name = '')
    @lv = lv
    @hp = hp
    @defense = defense
    @attack = attack
    @hit = hit
    @flee = flee
    @name = name
  end
end

File: Dart/lib/Enemy.dart
class Enemy {
  int Lv;
  int Hp;
  int Defense;
  int Attack;
  int Hit;
  int Flee;
  String Name;

  Enemy({int lv = 0, int hp = 0, int defense = 0, int attack = 0, int hit = 0, int flee = 0, String name = ''}) {
    Lv = lv;
    Hp = hp;
    Defense = defense;
    Attack = attack;
    Hit = hit;
    Flee = flee;
    Name = name;
  }
}